docarray/array/doc_list/doc_list.py

```diff
@@ -337,8 +337,14 @@ def __class_getitem__(cls, item: Union[Type[BaseDoc], TypeVar, str]):
 
         if isinstance(item, type) and safe_issubclass(item, BaseDoc):
             return AnyDocArray.__class_getitem__.__func__(cls, item)  # type: ignore
-        else:
-            return super().__class_getitem__(item)
+        if (
```

Comment:
should we better have an `elif`?

Reply:
Since each condition has a terminating condition from the function wouldn't multiple `if`'s be similar to `elif`? If there was a scenario like below an `elif` would definitely make it much better. In the above case, we check the instance of `item` and based on the condition we return a type or raise an exception. But if the code base follows `if ... elif ... else` blocks I would love to keep it uniform so that it is easy to maintain
```
computation
if condn1:
   computation = value1
elif condn2:
   computation = value2
else:
   computation = value3

return computation

```